web-ng/src/app/shared/converters/firebase-data-converter.ts

```diff
@@ -456,6 +457,10 @@ export class FirebaseDataConverter {
     };
   }
 
+  static aclToJs(acl: Map<string, Role>): {} {
```

Comment:
do you think it would be good to add test to converter?

Reply:
I do! We're still punting on tests for now, but we'll add this once we get an end-to-end demo working.